File: src/main/java/me/ruslanys/vkmusic/entity/Property.java
package me.ruslanys.vkmusic.entity;

import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;

/**
 * @author Ruslan Molchanov (ruslanys@gmail.com)
 */
@Data
@NoArgsConstructor

@Entity
public class Property {

    @Id
    private String id;

    @Column(nullable = false)
    private String json;

    public Property(String key, String json) {
        this.id = key;
        this.json = json;
    }

}

File: src/main/kotlin/me/ruslanys/vkmusic/entity/Property.kt
package me.ruslanys.vkmusic.entity

import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.Id

/**
 * @author Ruslan Molchanov (ruslanys@gmail.com)
 */
@Entity
class Property(
        @Id
        var id: String,

        @Column(nullable = false)
        var json: String
)
